packages/react-interactions/accessibility/src/FocusList.js

```diff
@@ -117,25 +127,34 @@ export function createFocusList(scope: ReactScope): Array<React.Component> {
           const listProps = list && list.getProps();
           if (list !== null && listProps.type === 'list') {
             const portrait = listProps.portrait;
-            switch (event.key) {
-              case 'Tab': {
-                const tabScope = getListProps(currentItem).tabScopeRef.current;
-                if (tabScope) {
-                  const activeNode = document.activeElement;
-                  const nodes = tabScope.getScopedNodes();
-                  for (let i = 0; i < nodes.length; i++) {
-                    const node = nodes[i];
-                    if (node !== activeNode) {
-                      setElementCanTab(node, false);
-                    } else {
-                      setElementCanTab(node, true);
-                    }
+            const key = event.key;
+
+            if (key === 'Tab') {
```

Comment:
Given that "shift" is a modifier, and it impacts "Tab" (reverses the direction) it seems odd that we check for `key === 'Tab'` _before_ bailing out if modifiers aren't allowed. Is this a bug?

Reply:
The modifiers logic is only for the keyboard arrow keys, not for Tab.

Reply:
I see. That wasn't super clear to me just looking at the code and going off of the name of the prop. Maybe some inline comment clarifying the intent could be helpful?